balancer/frontend-v2
# zkEVM promo banner

# Description

- New promo banner for zkEVM, which appears on all non-zkEVM networks.
- Updated the other Polygon chain name from `Polygon` to `Polygon PoS`.

## Type of change

- [x] Other: New promo banner

## How should this be tested?

A new promo banner should appear on all networks, except on the `/zkevm` pools list page.

<img width="1197" alt="CleanShot 2023-06-12 at 22 13 20@2x" src="https://github.com/balancer/frontend-v2/assets/1121139/d8605f83-df0f-4f17-a52e-b547c74d7039">

The new network name of `Polygon PoS` should appear in Networks:

<img width="255" alt="CleanShot 2023-06-12 at 22 27 21@2x" src="https://github.com/balancer/frontend-v2/assets/1121139/dc807401-86dd-409b-986d-f733f3b8d021">

There is a chance that this may break logic if we are specifically targeting the network based on chain names anywhere in the app. Please review and consider any place which may break upon a change in `ChainName`.

## Checklist:
